benchmarks/overrides_benchmark/README.md

```diff
@@ -0,0 +1,39 @@
+# `__torch_function__` micro-benchmarks
+
+This benchmark suite provides a systemic way to measure the performance of `__torch_function__` overhead.
+
+## Getting started
+### Initial Setup
+Install `py-spy` by doing:
+
+```bash
+pip install py-spy
+```
+
+Note that more extensive documentation on using `py-spy` is available in `CONTRIBUTING.md`.
+
+### Running the benchmark
+Run one of the following commands in the terminal, with the working directory being `${PYTORCH_CLONE_DIR}/benchmarks/overrides_benchmark`:
+
+```bash
+# Benchmark all the cases
+python bench.py
+
+# Flame graph pertaining to each case.
+py-spy record -o tensor.svg --native -- python pyspybench.py Tensor
+py-spy record -o subtensor.svg --native -- python pyspybench.py SubTensor
+py-spy record -o overridden.svg --native -- python pyspybench.py WithTorchFunction
+```
+
+Here is a brief overview of what the results should look like, if run correctly:
+
+* Overhead for `torch` functions when run on `torch.Tensor` objects is close to 2 μs.
```

Comment:
I think quoting exact numbers like this may not be that helpful, won't the exact normalization depend on which CPU was used?

Reply:
I'll change the wording here, but @rgommers requested we put numbers in.

Reply:
I think I said "on the order of 2 us". Which is useful imho. Numbers don't vary _that_ widely, typically by 20-40%, only if you have a really old CPU do you get larger times.

The point is to somehow express that we're looking for O(100 ns) numbers that are really hard to benchmark but still significant. Doing that without mentioning any numbers at all will be more convoluted.